source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst

```diff
@@ -42,9 +47,13 @@ In almost every event we see at least one robot where a not properly secured bat
   :alt: Applying a twisting force to the second cable on the breaker.
   :width: 350
 
-Apply a twisting force onto the cable to rotate the harness.  If you are successful then the screw is not tight enough.  Split washers might help here, but in the mean time, these require checking every few matches.
+Apply a strong twisting force to try to rotate the crimped lug.  If the lug rotates then the nut is not tight enough.  After tightening the nut, retest by once again trying to rotate the lug.
+
+The original nut has a star locking feature, which can wear out over time: these may require checking every few matches, especially if your robot-side battery connector is not attached to the chassis.
+
+The nut is normally a relatively uncommon 1/4-28 thread: ensure this is correct if the nut is replaced.
 
-Because the metal is just molded into the case, every once in awhile you will break off the bolt, ask any veteran team and they’ll tell you they go through a number of these every few seasons.  After tightening the nut, retest by once again trying to twist the cable.
+Because the metal is just molded into the case, every once in awhile you may break off the stud.  Many veteran teams start each season with a fresh main breaker, and carry spares.
```

Comment:
We've also had vibration damage and fatigue of the latch plastic lead to a main breaker that trips itself off under hard impacts. Agreed re: causality, I've broken the causality around replacing the breaker into it's own section.

Reply:
@Kevin-OConnor I think this is our last dangling conversation

Reply:
Maybe just change "Many" to "Some"?  I agree that this doesn't seem prevalent enough to warrant "many".

Reply:
The language here now with "may" and a more generic statement of many teams starting with a fresh breaker (not necessarily because of the previous) seems fine to me.